perllib/FixMyStreet/Cobrand/Peterborough.pm

```diff
@@ -489,6 +489,8 @@ sub look_up_property {
 
     my %premises = map { $_->{uprn} => $_ } @$premises;
 
+    my $attributes = $self->property_attributes($uprn);
```

Comment:
My one slight hesitation is that this adds another blocking Bartec API request to the "Your bin days" page, for a total of seven. It does appear to be the fastest, taking ~70ms locally, so perhaps it doesn't matter but thought it worth mentioning.

Reply:
Yeah, I did wonder about that, but I couldn't see another way to know whether to show the report/request links on the bin days page. Guess we could leave the links showing and only if you clicked through would it not have them/greyed them out, but that didn't seem ideal.